<launch>
    <!-- Bring up the tilt_laser aware version of move_base -->
<!--<node pkg="pr2_move_base" name="pr2_move_base_node" type="pr2_move_base.py" machine="c2" output="screen">
    <remap from="move_base/goal" to="move_base_local/goal" />
    <remap from="move_base/result" to="move_base_local/result" />
    <remap from="move_base/feedback" to="move_base_local/feedback" />
    <remap from="move_base/status" to="move_base_local/status" />
    <remap from="move_base/cancel" to="move_base_local/cancel" />
    <remap from="pr2_move_base/goal" to="pr2_move_base_local/goal" />
    <remap from="pr2_move_base/result" to="pr2_move_base_local/result" />
    <remap from="pr2_move_base/feedback" to="pr2_move_base_local/feedback" />
    <remap from="pr2_move_base/status" to="pr2_move_base_local/status" />
    <remap from="pr2_move_base/cancel" to="pr2_move_base_local/cancel" />
  </node>-->

  <!-- Throttle the voxel grid that is being published for rviz -->
  <node ns="move_base_local_node/local_costmap" pkg="topic_tools" type="throttle" name="voxel_grid_throttle" machine="c2" args="messages voxel_grid 3.0 voxel_grid_throttled" />

  <node pkg="move_base" type="move_base" respawn="false" name="move_base_local_node" machine="c2">
    <remap from="odom" to="base_odometry/odom" />
    <remap from="cmd_vel" to="navigation/cmd_vel" />

    <!-- Change the name of the action that we're using -->
    <remap from="move_base" to="move_base_local" />
    <remap from="move_base_simple" to="move_base_local_simple" />

    <!-- Use the dwa local planner for the PR2 -->
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />


    <rosparam file="$(find book_stacking)/config/move_base_params.yaml" command="load" />
    <rosparam file="$(find book_stacking)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find book_stacking)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find book_stacking)/config/dwa_local_planner.yaml" command="load" ns="DWAPlannerROS" />
    <rosparam file="$(find book_stacking)/config/recovery_behaviors.yaml" command="load" />
    <rosparam file="$(find book_stacking)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find book_stacking)/config/local_costmap_params.yaml" command="load"  />


<!--
    <rosparam file="$(find pr2_navigation_config)/move_base/move_base_params.yaml" command="load" />
    <rosparam file="$(find pr2_navigation_config)/move_base/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find pr2_navigation_config)/move_base/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find pr2_navigation_config)/move_base/dwa_local_planner.yaml" command="load" ns="DWAPlannerROS" />
    <rosparam file="$(find pr2_navigation_config)/move_base/recovery_behaviors.yaml" command="load" />

    <rosparam file="$(find pr2_navigation_local)/config/move_base_params.yaml" command="load" />
    <rosparam file="$(find pr2_navigation_local)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find pr2_navigation_local)/config/local_costmap_params.yaml" command="load"  />
    <rosparam file="$(find pr2_navigation_local)/config/dwa_local_planner.yaml" command="load"  ns="DWAPlannerROS" />-->



  </node>
</launch>
